### ISSUE TITLE
isomorphic fetch is broken.

### ISSUE THREAD

**Issue by ryan-roemer** [FormidableLabs/converter-flux#18]
```
import fetch from "isomorphic-fetch";
```

is _incorrect_. https://github.com/matthew-andrews/isomorphic-fetch/pull/20

Should be:

```
import "isomorphic-fetch";
```

**Comment by ryan-roemer** [comment]
/cc @baer 

(I'm fixing in WIP misc cleanup branch)

**Comment by baer** [comment]
That issue gives me pause about that lib. Anyhow looks like it was fixed in the latest release. Updating package.json should do the trick:

https://github.com/matthew-andrews/isomorphic-fetch/pull/20#issuecomment-121962169

**Comment by ryan-roemer** [comment]
It shouldn't give us pause about the library. The documented use on the README is polyfilling; https://github.com/matthew-andrews/isomorphic-fetch#usage

```
require('isomorphic-fetch');
```

The _existing_ code here had polyfilling and worked just fine. This diff broke things:

``` diff
diff --git a/client/utils/api.js b/client/utils/api.js
index ada791c..ee74258 100644
--- a/client/utils/api.js
+++ b/client/utils/api.js
@@ -1,8 +1,8 @@
 /**
  * Fetch data from rest API.
  */
-require("es6-promise").polyfill();
-require("isomorphic-fetch");
+import Promise from "bluebird";
+import fetch from "isomorphic-fetch";

 let api = {
   BASE_URL: "",
```

**Comment by baer** [comment]
I incorrectly assumed that the library worked like many others in that I could use it without relying on import/require for the side effect of attaching things to `window`. I was mistaken but it looks like
1. I'm not 100% clear on it but the above syntax looks to be available in 2.1.x
2. You can still use the es6 style import and do `import 'isomorphic-fetch'`

**Comment by ryan-roemer** [comment]
> I'm not 100% clear on it but the above syntax looks to be available in 2.1.x

Yes. `import fetch from "isomorphic-fetch";` should work now. The awkward thing is that even with the new bugfix is that calling `require("isomorphic-fetch")` will **always** perform a polyfill. My thought is leave it as an explicit polyfill to make this apparent...

> You can still use the es6 style import and do import 'isomorphic-fetch'

Yes. That is exactly my earlier bugfix https://github.com/FormidableLabs/converter-react/commit/7e577d284408c63ae232d6089c49456db62761f2 ;)
